lib/ghomperm.gi

```diff
@@ -1074,6 +1074,8 @@ InstallMethod( CompositionMapping2, "group hom. with perm group hom.",
     function( hom1, hom2 )
     local   prd,  stb,  levs,  S,t,i,oli;
 
+    Size(Source(hom2));
+    Size(Range(hom2));
     stb := StructuralCopy( StabChainMutable( hom2 ) );
```

Comment:
A comment why the `Size` computations are useful would be helpful.
I would have guessed that `StabChainMutable` will compute the orders of source and range as soon as the values are interesting.